data/json/bionics.json

```diff
@@ -64,34 +73,61 @@
     "id": "bio_armor_head",
     "type": "bionic",
     "name": { "str": "Alloy Plating - head" },
-    "description": "The flesh on your head has been surgically replaced by alloy plating, protecting both your head and jaw regions.",
-    "occupied_bodyparts": [ [ "head", 5 ], [ "mouth", 1 ] ],
-    "bash_protec": [ [ "head", 3 ] ],
-    "cut_protec": [ [ "head", 3 ] ],
-    "bullet_protec": [ [ "head", 3 ] ],
-    "flags": [ "BIONIC_NPC_USABLE", "BIONIC_SHOCKPROOF" ]
+    "description": "The skin and flesh on your head has been surgically replaced by alloy plating, protecting both your head and jaw regions from physical and chemical injuries at the cost of flexibility.",
+    "occupied_bodyparts": [ [ "head", 6 ], [ "mouth", 2 ] ],
+    "bash_protec": [ [ "head", 7 ] ],
+    "cut_protec": [ [ "head", 6 ] ],
+    "bullet_protec": [ [ "head", 6 ] ],
+    "fire_protec": [ [ "head", 2 ] ],
+    "acid_protec": [ [ "head", 2 ] ],
+    "env_protec": [ [ "head", 2 ] ],
+    "encumbrance": [ [ "head", 10 ] ],
+    "wet_protection": [ { "part": "head", "ignored": 4 } ],
+    "covered_bodyparts": [ "head" ],
+    "no_bleed_chance": 0.25,
+    "no_bite_chance": 0.33,
+    "cant_uninstal_msg": "The Alloy Plating is part of %s body now.  Removing it would leave %s disabled.",
+    "flags": [ "BIONIC_NPC_USABLE", "BIONIC_SHOCKPROOF", "BIONIC_CANT_UNINSTAL", "BIONIC_CANT_COMPRESS" ]
```

Comment:
```suggestion
    "cant_uninstall_msg": "The Alloy Plating is part of %s body now.  Removing it would leave %s disabled.",
    "flags": [ "BIONIC_NPC_USABLE", "BIONIC_SHOCKPROOF", "BIONIC_CANT_UNINSTALL", "BIONIC_CANT_COMPRESS" ]
```